examples/multiple_viewers_.py

```diff
@@ -13,12 +13,13 @@
 
 # add the image
 photographer = data.camera()
-viewer_a = napari.view_image(photographer, name='photographer')
+viewer = napari.Viewer()
+layer_a = viewer.add_image(photographer, name='photographer')
 
 # add the image in a new viewer window
 astronaut = data.astronaut()
-# Also view_path, view_shapes, view_points, view_labels etc.
-viewer_b = napari.view_image(astronaut, name='astronaut')
+# Also view_path, add_shapes, add_points, add_labels etc.
```

Comment:
In this example a second viewer needs to be created so that there are two viewer windows `viewer_a` and `viewer_b. I believe the example's purpose is to demonstrate that multiple viewers are possible.